test/runner.zig

```diff
@@ -255,19 +255,28 @@ pub fn main() !void {
                 continue;
             }
 
-            if (macro.tokens.len != expected_count) {
-                fail_count += 1;
-                progress.log(
-                    \\EXPECTED_ERRORS missing errors, expected {d} found {d},
-                    \\=== actual output ===
-                    \\{s}
-                    \\
-                    \\
-                , .{ macro.tokens.len, expected_count, m.buf.items });
-                continue;
+            {
+                var count: u32 = 0;
+                for (macro.tokens) |tok| switch (tok.id) {
+                    .whitespace => {},
```

Comment:
Looks like the current method of using `#define EXPECTED_TOKENS` won't work. Either we'll just not test for whitespace or create a different way to test for preprocessor output.

Reply:
For preprocessing an option could be to have the test file and the result file (i.e. output of -E) separated, and the tester would check for byte to byte similarity of the two. It's not as convenient as to what there is now but it could work.

Reply:
Hmm, I'll try something like that.